lib/DateTime.pm

```diff
@@ -1365,6 +1369,7 @@ sub jd { $_[0]->mjd + 2_400_000.5 }
         my $self    = shift;
         my $pattern = shift;
 
+        ## no critic (ontrolStructures::ProhibitCStyleForLoops)
```

Comment:
IIRC if Perl::Critic doesn't see something it understands, it turns into `## no critic ()` which is just `## no critic`. I remember having the same problem.